test/e2e/tests/request-queuing/ui.spec.js

```diff
@@ -199,13 +204,10 @@ describe('Request-queue UI changes', function () {
           WINDOW_TITLES.ExtensionInFullScreenView,
         );
 
-        await driver.clickElement('[data-testid="sort-by-networks"]');
-        await driver.clickElement({
-          text: 'Custom',
-          tag: 'button',
-        });
+        const networkManager = new NetworkManager(driver);
```

Comment:
Cleanup test as network manager logic has changed.
Also using NetworkManager page object instead of custom driver calls.